### PR TITLE
Add presenter get user related models

### PR DESCRIPTION
**Pull request by reiterl**
Add some tests.
Perhaps the permission code is not optimal now.

### CODE REVIEW

**Comment by reiterl** [comment]
We discussed in a meeting about this issue. It should be changed.

**Comment by reiterl** [comment]
I updated the permission code and the tests. I also added some new tests for the permission check.

File: openslides_backend/presenter/get_user_related_models.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        committees = (
+            self.datastore.get_many([gmr]).get(Collection("committee"), {}).values()
+        )
+        for committee in committees:
+            committee_id = committee["id"]
+            user2 = self.datastore.get(
+                FullQualifiedId(Collection("user"), user_id),
+                [f"committee_${committee_id}_management_level"],
+            )
+            if (
```
Comment (r-peschke) [inline openslides_backend/presenter/get_user_related_models.py:100]: You don't need the if-block and the variable return_committees. See the changed/clarified requirements in [wiki](https://github.com/OpenSlides/OpenSlides/wiki/get_user_related_models#logic).

File: openslides_backend/presenter/get_user_related_models.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+            committees_data.append(
+                {
+                    "id": committee_id,
+                    "name": committee.get("name", ""),
+                    "cml": user2.get(f"committee_${committee_id}_management_level", ""),
+                }
+            )
+        if return_committees:
+            return committees_data
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/get_user_related_models.py:116]: Done.

File: openslides_backend/presenter/get_user_related_models.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            return []
+        gmr = GetManyRequest(
+            Collection("committee"), user["committee_ids"], ["id", "name"]
+        )
+        committees = (
+            self.datastore.get_many([gmr]).get(Collection("committee"), {}).values()
+        )
+        for committee in committees:
+            committee_id = committee["id"]
+            user2 = self.datastore.get(
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/get_user_related_models.py:96]: Read all permissions in one datastore access.

File: tests/system/presenter/test_get_user_related_models.py [hunk 4]
```diff
@@ -0,0 +1,209 @@
+from openslides_backend.permissions.management_levels import OrganizationManagementLevel
+from openslides_backend.permissions.permissions import Permissions
+
+from .base import BasePresenterTestCase
+
+
+class TestGetUserRelatedModels(BasePresenterTestCase):
```
[the remark this answers is not in the document]
Comment (reiterl) [reply tests/system/presenter/test_get_user_related_models.py:7]: Done.

**Pull request merged**